### PR TITLE
[Container app] az containerapp sessionpool create: Fix default container name to be lower case letters

### PR DESCRIPTION
**Pull request by smzeng**
---
image

Fix the default container name scenario.
when no container name is provided (param: --container-name) we default to the sessionpool name (param: --name). However sessionpool name accepts uppercase letters but --container-name does not. Add converting container name to lower case.

not a BUILD blocking PR because user can use --container-name parameter to specify container name or change sessionpool name to lowercase.

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Review by xwang971 — APPROVED** [review]

**Review by sanchitmehta — APPROVED** [review]

**Review by Greedygre — APPROVED** [review]

File: src/containerapp/HISTORY.rst [hunk 1]
```diff
@@ -4,6 +4,7 @@ Release History
 ===============
 upcoming
 ++++++
+* 'az containerapp sessionpool create': fix default container name to be lower case letters
```
Comment (Greedygre) [inline src/containerapp/HISTORY.rst:7]: * 'az containerapp sessionpool create': Fix default container name to be lower case letters

**Comment by Greedygre** [comment]
Recommend title:
[Container app] `az containerapp sessionpool create`: Fix default container name to be lower case letters

**Review by Greedygre — APPROVED** [review]

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**